homeassistant/components/shopping_list.py

```diff
@@ -15,9 +15,11 @@
 from homeassistant.util.json import load_json, save_json
 from homeassistant.components import websocket_api
 
+ATTR_LIST_ID = 'list_id'
 ATTR_NAME = 'name'
 
 DOMAIN = 'shopping_list'
+DEFAULT_LIST = 'system-inbox'
```

Comment:
Can we rename this `LIST_SYSTEM_INBOX` ? We should name things to what they are, not how they are used, because it can change how it's used in the future. It's like `report_final_final_final_v2.pdf` 😉